docs/markdown/DevGuide_Data_Memory.md

```diff
@@ -0,0 +1,85 @@
+# Data Structures
+In Inviwo the main core data structures (`Volume`, `Image`, `Layer`, `Mesh`, `Buffer`) use a pattern of handles and representations. The volume data structure for example has a handle class called `Volume`. The handle class by itself does not have any data, it only stores metadata, and a list of representations. The representations is where the actual data is stored. The following first explains the basic data structures and how they relate, then shows what different representations exist and how to use them.
+
+## The different structures
+In Inviwo, the `Volume` class wraps volumetric data, more specifically data in a structured grid. This corresponds to what you would expect from a rank 3 tensor (or rank 4 if you have a multi-channel volume).
```

Comment:
I am not an expert (even though I should be) but rank 4 tensors would not describe multi-channel volumes. They could describe time-dependent scalar fields, though. I have no idea how to describe multi-channel volumes in terms of tensors because afaik, the entries in the tensors are scalars.

Correct me if I'm wrong.

Reply:
Lets just not involve tensors in this

Reply:
Always the best solution :)